stockflux-securities-master/public/app.dev.json

```diff
@@ -14,12 +14,13 @@
         "minWidth": 850,
         "maxWidth": 850,
```

Comment:
Do we want to set Max Width and Height, I'd imagined 'Maximize' should be allowed for this as moving forward we may have a number of properties on each security.